content/en/security/cloud_siem/investigate_security_signals.md

```diff
@@ -50,36 +50,93 @@ To view your signals by MITRE ATT&CK Tactic and Technique:
 
 1. Navigate to [Cloud SIEM][5].
-1. Click **Signals**.
+1. Click the **Signals** tab at the top of the page.
 1. Click on a security signal from the table.
-1. To assign a signal to yourself or another Datadog user, click the user profile icon with the plus sign in the top left corner of the signal side panel.
-{{< img src="security/security_monitoring/investigate_security_signals/profile_icon.png" alt="The profile icon next to the triage status" style="width:45%;" >}}
-1. To update the triage status of the security signal, navigate to the top left corner of the signal side panel and select the status you want from the dropdown menu. The default status is `OPEN`.
-{{< img src="security/security_monitoring/investigate_security_signals/triage_status.png" alt="The profile icon next to the triage status" style="width:35%;" >}}
-    - **Open**: Datadog Security triggered a detection based on a rule, and the resulting signal is not yet resolved.
-    - **Under Review**: During an active investigation, you can switch the signal state to **Under Review**. From the **Under Review** state, you can move the signal state to **Archived** or **Open** as needed.
-    - **Archived**: When the detection that caused the signal has been resolved, you can transition it to the **Archived** state. If an archived issue resurfaces, or if further investigation is necessary, a signal can be changed back to an **Open** state within 30 days of being created.
+1. In the **What Happened** section, see the logs that matched the query. Hover over the query to see the query details.
+    - You can also see specific information like username or network IP. In **Rule Details**, click the funnel icon to create a suppression rule or add the information to an existing suppression. See [Create suppression rule][11] for more details.
+1. In the **Next Steps** section:   
+  a. Under **Triage**, click the dropdown to change the triage status of the signal. The default status is `OPEN`.
+      - `Open`: Datadog Security triggered a detection based on a rule, and the resulting signal is not yet resolved.
+      - `Under Review`: During an active investigation, change the triage status to `Under Review`. From the `Under Review` state, you can move the status to `Archived` or `Open` as needed.
+      - `Archived`: When the detection that caused the signal has been resolved, update the status to `Archived`. When a signal is archived, you can give a reason and description for future reference. If an archived issue resurfaces, or if further investigation is necessary, the status can be changed back to `Open`. All signals are locked 30 days after they have been created.</ul>
+  b. Click **Assign Signal** to assign a signal to yourself or another Datadog user.
+  c. Under **Take Action**, you can create a case, declare an incident, edit suppressions, or run workflows. Creating a case automatically assigns the signal to you and sets the triage status to `Under Review`.
```

Comment:
Something is wrong with the formatting of this item. It appears inline after b. instead of on its own line.

Reply:
Thanks for catching that! Fixed it.